src/test/kotlin/com/gamingbacklog/api/gamingbacklogapi/unit/services/LibraryServiceTests.kt

```diff
@@ -133,6 +136,53 @@ class LibraryServiceTests {
     }
   }
 
+  @Nested
+  @DisplayName("Tests for addToLibraries")
+  inner class AddToLibraries {
+    private val libraryIds = listOf(libraryId, libraryId2, libraryId3)
+    @Test
+    fun shouldAllSucceed() {
+      given(gameInstanceService.getSingle(gameId1)).willReturn(testGameInstance)
+      var count = 1
+      libraryIds.forEach {
+        saveMockLibraryDB(it, "${count++}", arrayListOf())
+      }
+      val result = libraryService.addToLibraries(libraryIds, gameId1)
+      assertEquals(MultiLibraryStatus.SUCCESS, result.libraryStatus)
+    }
+
+    @Test
+    fun shouldError_EmptyLibraryIds() {
+      given(gameInstanceService.getSingle(gameId1)).willReturn(testGameInstance)
```

Comment:
Is this mock necessary? Won't we error out before validating the gameId?

Reply:
yep 🤦‍♀️